lib/rubocop/cop/lint/to_json.rb

```diff
@@ -29,7 +29,7 @@ def on_def(node)
 
         def autocorrect(node)
           lambda do |corrector|
-            corrector.insert_after(node.loc.name, '(_opts)')
+            corrector.insert_after(node.loc.name, '(*args)')
```

Comment:
`def to_json(*args)` has an offense of `Lint/UnusedMethodArgument` cop if `args` is not used.
So I think it should be named `_args`.

Reply:
Sure! I updated it and added a source code comment.